### PR TITLE
Update cmctl docs in preparation for v2 release

### PR DESCRIPTION
**Pull request by inteon**
Update the installation instructions to point to the new repo.

### CODE REVIEW

File: content/docs/reference/cmctl.md [hunk 1]
```diff
@@ -34,17 +34,17 @@ This will also install shell completion.
...
+[cmctl GitHub releases page](https://github.com/cert-manager/cmctl/releases).
 In order to use `cmctl` you need its binary to be accessible under
 the name `cmctl` in your `$PATH`.
 Run the following commands to set up the CLI. Replace OS and ARCH with your
 systems equivalents:
 
 ```console
-OS=$(go env GOOS); ARCH=$(go env GOARCH); curl -fsSL -o cmctl.tar.gz https://github.com/cert-manager/cert-manager/releases/latest/download/cmctl-$OS-$ARCH.tar.gz
-tar xzf cmctl.tar.gz
+OS=$(go env GOOS); ARCH=$(go env GOARCH); curl -fsSL -o cmctl https://github.com/cert-manager/cmctl/releases/latest/download/cmctl_${OS}_${ARCH}
```
Comment (inteon) [inline content/docs/reference/cmctl.md:46]: This does not yet work because there is no `latest` version yet.

Comment (inteon) [reply content/docs/reference/cmctl.md:46]: Works now that v2.0.0 has been released.

File: content/docs/reference/cmctl.md [hunk 2]
```diff
@@ -34,17 +34,17 @@ This will also install shell completion.
...
+chmod +x cmctl
 sudo mv cmctl /usr/local/bin
 ```
```
Comment (maelvls) [inline content/docs/reference/cmctl.md:49]: I noticed I can now install `cmctl` using `go install`. It think it's an option that lots of people would be happy knowing about:

~~~suggestion
```

Alternatively, you can install `cmctl` using `go`:

```bash
go install github.com/cert-manager/cmctl/v2@latest
```
~~~

File: content/docs/reference/cmctl.md [hunk 3]
```diff
@@ -34,20 +34,26 @@ This will also install shell completion.
...
 the name `cmctl` in your `$PATH`.
 Run the following commands to set up the CLI. Replace OS and ARCH with your
 systems equivalents:
 
 ```console
-OS=$(go env GOOS); ARCH=$(go env GOARCH); curl -fsSL -o cmctl.tar.gz https://github.com/cert-manager/cert-manager/releases/latest/download/cmctl-$OS-$ARCH.tar.gz
-tar xzf cmctl.tar.gz
+OS=$(go env GOOS); ARCH=$(go env GOARCH); curl -fsSL -o cmctl https://github.com/cert-manager/cmctl/releases/latest/download/cmctl_${OS}_${ARCH}
+chmod +x cmctl
 sudo mv cmctl /usr/local/bin
```
Comment (maelvls) [inline content/docs/reference/cmctl.md:48]: `install` combines `chmod` + `mv`:

```suggestion
sudo install cmctl /usr/local/bin
```

**Review by maelvls — APPROVED** [review]
/approve
/lgtm

File: content/docs/reference/cmctl.md (hunk 3 above)
Comment (inteon) [reply content/docs/reference/cmctl.md:48]: how universal is this command (linux, osx ...)?

**Comment by inteon** [comment]
I'm not very familiar with the "install" command.
Let's leave that change to another PR.
/unhold

**Pull request merged**